### PR TITLE
Advertise kubeapps

### PR DESCRIPTION
**Pull request by earnubs**
Place a small advert for kubeapps in chart details linking to the
kubeapps github repo (rather than kubeapps.com, which isn't considered
a useful entry point for hub users).

image

### CODE REVIEW

File: src/app/chart-details/chart-details-usage/chart-details-usage.component.html [hunk 1]
```diff
@@ -26,4 +26,17 @@ <h1>Install</h1>
...
+    <img
+      alt="kubeapps"
+      class="kubeapps-logo"
+      src="/assets/images/kubeapps-black-on-white.svg"
+      width="110"
+      />
+    Easily deploy and manage {{ chart.attributes.name }} in your
+    Kubernetes cluster with Kubeapps.<br />
+    <a href="https://github.com/kubeapps/kubeapps/blob/master/docs/user/getting-started.md" >
+      Get Started on GitHub
```
Comment (earnubs) [inline src/app/chart-details/chart-details-usage/chart-details-usage.component.html]:
```suggestion
      Get Started on GitHub.
```

**Comment by andresmgot** [comment]
Can we reduce the padding of the `Need Helm` link? It seems to much. 

Also the padding-bottom of the logo is also a bit bigger, I think we can reduce that a bit? Related to that, are you using a black and white logo for something specific? Maybe the one with color is a bit more catchy.

In any case, let's wait for @prydonius input here.

**Comment by prydonius** [comment]
> Also the padding-bottom of the logo is also a bit bigger, I think we can reduce that a bit?

+1

I like the use of the black and white logo as it feels less noisy colour-wise.

File: src/app/chart-details/chart-details-usage/chart-details-usage.component.html [hunk 2]
```diff
@@ -26,4 +26,17 @@ <h1>Install</h1>
...
+    <img
+      alt="kubeapps"
+      class="kubeapps-logo"
+      src="/assets/images/kubeapps-black-on-white.svg"
+      width="110"
+      />
+    Easily deploy and manage {{ chart.attributes.name }} in your
+    Kubernetes cluster with Kubeapps.<br />
+    <a href="https://github.com/kubeapps/kubeapps/blob/master/docs/user/getting-started.md" >
+      Get Started on GitHub.
```
Comment (prydonius) [inline src/app/chart-details/chart-details-usage/chart-details-usage.component.html]: Is the full stop necessary, it looks a little out-of-place to me?

Comment (earnubs) [inline src/app/chart-details/chart-details-usage/chart-details-usage.component.html]:
```suggestion
      Get Started on GitHub
```

Comment (earnubs) [inline src/app/chart-details/chart-details-usage/chart-details-usage.component.html]:
```suggestion
      Get Started on GitHub
```

File: src/app/chart-details/chart-details-usage/chart-details-usage.component.scss [hunk 3]
```diff
@@ -23,12 +23,26 @@ mat-tab-body {
...
+    margin-top: 2em;
+    text-align: center;
+  }
+
+  .kubeapps-logo {
+    display: block;
+    width: 120px;
+    display: block;
+    margin: 0 auto;
+    padding: 0 0 1em;
```
Comment (earnubs) [inline src/app/chart-details/chart-details-usage/chart-details-usage.component.scss]:
```suggestion
    padding: 0 0 0.6em;
```

File: src/app/chart-details/chart-details-usage/chart-details-usage.component.scss [hunk 4]
```diff
@@ -23,12 +23,26 @@ mat-tab-body {
...
-    &.help-link {
-      font-size: 0.8em;
-      text-align: center;
-      margin-top: 20px;
-      margin-bottom: 0;
-    }
+  .help-link {
+    font-size: 0.8em;
+    text-align: center;
+    margin-top: 20px;
```
Comment (earnubs) [inline src/app/chart-details/chart-details-usage/chart-details-usage.component.scss]:
```suggestion

```

Comment (earnubs) [reply src/app/chart-details/chart-details-usage/chart-details-usage.component.scss]: Reduce spacing around help link, per review.

**Comment by earnubs** [comment]
image

**Comment by arapulido** [comment]
I think we can merge this now, and deploy it

**Pull request merged**